Successfully updated:
Created Frame node P4FsOwcEC
Created Frame node xWZnFQnwL
Created Text node IHnFcNsYz
Created Text node YGO9Ololw
Created Frame node yUb9Zfdzc
Updated attributes for node P4FsOwcEC
Updated attributes for node xWZnFQnwL
Updated text for node IHnFcNsYz
Updated attributes for node IHnFcNsYz
Updated text for node YGO9Ololw
Updated attributes for node YGO9Ololw
Updated attributes for node yUb9Zfdzc
Reordered node xWZnFQnwL within parent P4FsOwcEC to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -3482,20 +3482,73 @@
       Header Text
     </HeaderText>
   </Frame>
   <BodyContentGoesHere
       nodeId="KlUEXbXs1"
       width="fit-content"
       height="fit-content"
       font="Inter"
   >
     Body content goes here
   </BodyContentGoesHere>
   <Frame
       nodeId="SExft2gq8"
       width="100%"
       height="1fr"
       <!-- background color string or project color style path (if starts with /) -->
       backgroundColor="rgb(255, 255, 255)"
       borderRadius="4px"
    />
 </Stack>
+<Stack
+    nodeId="P4FsOwcEC"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="xWZnFQnwL"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="IHnFcNsYz"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="YGO9Ololw"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="yUb9Zfdzc"
+      width="100%"
+      height="1fr"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>


Note: Set zoomIntoView=false if you want to use Framer app while MCP is working.